Update minikube.md (#2043)
api-platform/docs: deployment/minikube.md

The previous command starts Minikube with a Docker registry (we'll use it in the next step) and with the Kubernetes dashboard.

Finally, [install Helm](https://helm.sh/docs/intro/install/). We'll use it to deploy the application in the cluster thanks to the chart provided in the API Platform distribution.

## Building and Pushing Docker Images

> On GNU/Linux and macOS, to point your terminal to use the docker daemon inside minikube run this:

Reviewer: ```suggestion
On GNU/Linux and macOS, run the following command following command to point your terminal’s docker-cli to the Docker Engine inside minikube:
```

```console

>     eval $(minikube docker-env)

Reviewer: ```suggestion
eval $(minikube docker-env)
```

```

> Now any `docker` command you run in this current terminal will run against the docker inside minikube cluster. For detailed explanation and instructions for Windows [visit official minikube documentation](https://minikube.sigs.k8s.io/docs/handbook/pushing/#1-pushing-directly-to-the-in-cluster-docker-daemon-docker-env).

Reviewer: ```suggestion
Now any `docker` command you run in this current terminal will run against the Docker Engine inside the minikube cluster. For detailed explanation and instructions for Windows [visit official minikube documentation](https://minikube.sigs.k8s.io/docs/handbook/pushing/#1-pushing-directly-to-the-in-cluster-docker-daemon-docker-env).
```

Build the images in minikube:

```console

>     docker build -t localhost:5000/php api --target frankenphp_prod
>     docker build -t localhost:5000/pwa pwa --target prod

Reviewer: ```suggestion
docker build -t localhost:5000/php api --target frankenphp_prod
docker build -t localhost:5000/pwa pwa --target prod
```

```
    

> Then push the images in the registry installed in Minikube:

Reviewer: ```suggestion
Then push the images in the registry available in minikube:
```

```console

>     docker push localhost:5000/php
>     docker push localhost:5000/pwa

Reviewer: ```suggestion
docker push localhost:5000/php
docker push localhost:5000/pwa
```

```
    
## Deploying

Fetch Helm chart dependencies:

```console

>     helm repo add postgresql https://charts.bitnami.com/bitnami/
>     helm dependency build helm/api-platform

Reviewer: ```suggestion
helm repo add postgresql https://charts.bitnami.com/bitnami/
helm dependency build helm/api-platform
```

```
    
Finally, deploy the project using the Helm chart:

```console

>     helm install my-project helm/api-platform \
>       --set php.image.repository=localhost:5000/php \
>       --set php.image.tag=latest \
>       --set pwa.image.repository=localhost:5000/pwa \
>       --set pwa.image.tag=latest

Reviewer: ```suggestion
helm install my-project helm/api-platform \
  --set php.image.repository=localhost:5000/php \
  --set php.image.tag=latest \
  --set pwa.image.repository=localhost:5000/pwa \
  --set pwa.image.tag=latest
```
